.github/workflows/build.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        os: ['ubuntu-18.04', 'ubuntu-20.04']
+        os: ['ubuntu-18.04', 'ubuntu-20.04', 'ubuntu-22.04']
```

Comment:
+1, I think it would also be okay to remove `ubuntu-18.04` since it is now deprecated.

Reply:
👍